ui/public/i18n/language.json

```diff
@@ -177,7 +177,7 @@
     "more_info_about": "More info about",
     "virtualhost": "Virtual hosts",
     "Reverse_Proxy": "Reverse Proxy",
-    "path-and-virtual-host-rules": "If you want to create an URL path rule, the name field must start with a '/'. If you need to create a virtual host name rule, then the name field must match a virtual host name",
+    "path-and-virtual-host-rules": "If you want to create a resource path matching rule, then enter the resource path beginning with a '/'. If you want to create a web site name matching rule, then type the web site fully qualified domain name.",
```

Comment:
I prefer the old label

Reply:
- The old string says "URL path" rule. The manual says "resource path". I want to use the same wording.
- The old string says `virtual host` and that can be misleading. It already happened that people go and create a virtual host.